rma_refund_reason/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Add refund reason on rma refund creation process
```

Comment:
Perhaps saying also that they are configurable in the RMA operations?

Reply:
Done!